Document that component name has to be unique in a namespace (#298)
konflux-ci/docs: modules/ROOT/pages/getting-started.adoc

A Component CR describes the properties for an OCI artifact including the git repository where the artifact is from,
its latest built commit, initial build configuration parameters (i.e. Containerfile path), and any relationships to
other Components. The CR also contains a reference to its single owning Application.

> Component name has to be unique in a namespace, even though it would be in different application.

Reviewer: How about making a change like this to both places?
```suggestion
Component names must be unique in a namespace even components are in separate applications.
```

Author: changed
